_overviews/scala3-migration/tooling-tour.md

```diff
@@ -82,7 +82,7 @@ Full-fledged support is being worked on by the team at JetBrains.
 
 ### Scalafmt
 
-[Scalafmt](https://scalameta.org/scalafmt/) v3.0.0-RC1 supports both Scala 2.13 and Scala 3.
+[Scalafmt](https://scalameta.org/scalafmt/) v3.0.0-RC3 supports both Scala 2.13 and Scala 3.
```

Comment:
Shouldn't this just be v3.0.0?

Reply:
This is scalafmt, which has not released a v3.0.0 yet